assets/js/src/core/components/modal/form-modal/form-modal.styles.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { createStyles } from 'antd-style'
+
+export const useStyle = createStyles(({ token, css }) => {
+  return {
+    formModal: css`
+      .ant-space {
+        width: 100%;
```

Comment:
can we replace this with our w-full classes?

Reply:
done with: https://github.com/pimcore/studio-ui-bundle/pull/564/commits/d2381b8c0b27039dd3132357af9d918caed86800